app/release_metadata.json

```diff
@@ -1,8 +1,8 @@
 {
-  "version": "3.1.2-rc",
-  "stability": "rc",
+  "version": "3.2.0-alpha",
+  "stability": "alpha",
   "minimum_php_version": "7.2.21",
   "maximum_php_version": "7.4.99",
-  "minimum_mautic_version": "3.0.0-alpha",
+  "minimum_mautic_version": "3.1.0",
```

Comment:
We need `3.1.0` as a minimum version as that version introduced the `bin/console mautic:install` command used by CI